### PR TITLE
[ Xedra Evolved ]  Dreamsmith's Bed of Restful Repose

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
I was talking about how the various Xedra magic classes would handle conversion to wounds and Renech pointed out that Dreamsmith's could be completed right now.  So I did.

#### Describe the solution
Adds furniture and the item to create it then the recipe to create that.

#### Describe alternatives you've considered
None

#### Testing
TBD

### CODE REVIEW

File: data/mods/Xedra_Evolved/construction.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "required_skills": [ [ "fabrication", 1 ] ],
+    "time": "120 m",
+    "components": [
+      [ [ "dreamforged_bed_of_restful_repose", 1 ] ]
+    ],
+    "pre_special": "check_empty",
+    "post_terrain": "f_dreamforged_bed_of_restful_repose",
+    "activity_level": "LIGHT_EXERCISE"
+  }
+]
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/construction.json:16]:
```suggestion
]

```

File: data/mods/Xedra_Evolved/construction_group.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "type": "construction_group",
+    "id": "bed_upgrades",
+    "name": "Bed of Restful Repose"
+  }
+]
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/construction_group.json:7]:
```suggestion
]

```

File: data/mods/Xedra_Evolved/items/dreamsmith.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "flags": [ "NO_SALVAGE" ],
+    "price": 25000,
+    "price_postapoc": 1500,
+    "longest_side": "213 cm",
+    "volume": "570 L",
+    "weight": "57 kg",
+    "material": [ { "type": "scrap_dreamdross", "portion": 75 }, { "type": "wood", "portion": 25 } ],
+    "to_hit": -8
+  }
+]
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/dreamsmith.json:18]:
```suggestion
]

```

**Comment by MerryMooose** [comment]
the item form of this has in its description that it is folded to be about the size of a keyboard, however, its actually 570L. I have to wield this to take it with me on my deathmobile :(

**Pull request merged**